data-prepper-plugins/dynamodb-source-coordination-store/src/main/java/org/opensearch/dataprepper/plugins/sourcecoordinator/dynamodb/DynamoDbSourcePartitionItem.java

```diff
@@ -9,12 +9,16 @@
 import org.opensearch.dataprepper.model.source.coordinator.SourcePartitionStoreItem;
```

Comment:
Is there a test file for `DynamoDbSourcePartitionItem.java`?

Reply:
No it's just a model class with getters and setters. Has absolutely zero logic that's worthy of testing